custom_components/google_home/sensor.py

```diff
@@ -179,7 +179,9 @@ def state(self) -> str | None:
         next_alarm = device.get_next_alarm()
         return (
             next_alarm.local_time_iso
-            if next_alarm and next_alarm.status != GoogleHomeAlarmStatus.INACTIVE
+            if next_alarm
+            and next_alarm.status != GoogleHomeAlarmStatus.INACTIVE
+            and next_alarm.status != GoogleHomeAlarmStatus.MISSED
```

Comment:
How about a shorthand version here to keep the implementation consistent?

```suggestion
            and next_alarm.status not in (GoogleHomeAlarmStatus.INACTIVE, GoogleHomeAlarmStatus.MISSED)
```